app/src/main/res/layout/activity_bill.xml

```diff
@@ -18,323 +18,325 @@
+                                        app:srcCompat="@drawable/ic_cart_60dp"
+                                        tools:ignore="ContentDescription"/>
+                                </android.support.v7.widget.CardView>
+                            </android.support.design.widget.TextInputLayout>
+                        </LinearLayout>
                     </LinearLayout>
-                </LinearLayout>
-            </android.support.v7.widget.CardView>
-
-            <android.support.v7.widget.RecyclerView
-                android:layout_width="match_parent"
-                android:layout_height="0dp"
-                android:layout_weight="1"
-                android:padding="@dimen/dp_5"
-                bind:adapter="@{viewModel.adapter}"
-                bind:layoutManager="@{LayoutManager.linear()}"/>
+                </android.support.v7.widget.CardView>
 
-            <LinearLayout
-                android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                android:orientation="vertical">
-
-                <TextView
-                    android:layout_width="wrap_content"
-                    android:layout_height="wrap_content"
-                    android:layout_gravity="end"
-                    android:layout_marginRight="@dimen/dp_10"
-                    android:layout_marginEnd="@dimen/dp_10"
-                    android:text="@{viewModel.total + ``}"
-                    android:textColor="@color/color_green_accent_400"
-                    android:textSize="@dimen/sp_20"
-                    android:textStyle="bold"/>
-
-                <android.support.v7.widget.AppCompatButton
+                <android.support.v7.widget.RecyclerView
                     android:layout_width="match_parent"
-                    android:layout_height="wrap_content"
-                    android:layout_gravity="bottom"
-                    android:layout_marginBottom="@dimen/dp_10"
-                    android:layout_marginTop="@dimen/dp_10"
-                    android:background="@color/colorPrimary"
-                    android:onClick="@{() -> viewModel.onCreateBillClick()}"
-                    android:padding="@dimen/dp_12"
-                    android:text="@string/title_create"
-                    android:textColor="@android:color/white"/>
+                    android:layout_height="0dp"
+                    android:layout_weight="1"
+                    android:padding="@dimen/dp_5"
+                    bind:adapter="@{viewModel.adapter}"
+                    bind:layoutManager="@{LayoutManager.linear()}"/>
+
             </LinearLayout>
-        </LinearLayout>
-    </android.support.v4.widget.NestedScrollView>
+        </android.support.v4.widget.NestedScrollView>
+
+
+        <TextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="end|right"
```

Comment:
WARNING: [RtlHardcoded] Use "`end`" instead of "`right`" to ensure correct behavior in right-to-left locales
 Using `Gravity#LEFT` and `Gravity#RIGHT` can lead to problems when a layout is rendered in locales where text flows from right to left. Use `Gravity#START` and `Gravity#END` instead. Similarly, in XML `gravity` and `layout_gravity` attributes, use `start` rather than `left`.

For XML attributes such as paddingLeft and `layout_marginLeft`, use `paddingStart` and `layout_marginStart`. *NOTE*: If your `minSdkVersion` is less than 17, you should add *both* the older left/right attributes *as well as* the new start/right attributes. On older platforms, where RTL is not supported and the start/right attributes are unknown and therefore ignored, you need the older left/right attributes. There is a separate lint check which catches that type of error.

(Note: For `Gravity#LEFT` and `Gravity#START`, you can use these constants even when targeting older platforms, because the `start` bitmask is a superset of the `left` bitmask. Therefore, you can use `gravity="start"` rather than `gravity="left|start"`.)